src/components/search/History/SearchHistory.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Icon as UIIcon, List as UIList } from '@faststore/ui'
+import Button from 'src/components/ui/Button'
+import './SearchHistory.scss'
+import Link from 'src/components/ui/Link'
+import Icon from 'src/components/ui/Icon'
+import useSearchHistory from 'src/sdk/search/useSeachHistory'
+import { formatSearchState, initSearchState } from '@faststore/sdk'
+
+const doSearch = (term: string) => {
```

Comment:
Noticed we have this same function on the `SearchInput` component so I wonder we can how we can extract it to a single place.

https://github.com/vtex-sites/base.store/blob/157ac94f2ef63cba213bb78ffb4712145d5d4dec/src/components/common/SearchInput/SearchInput.tsx#L24-L30

Reply:
Nice catch!
I think when we have all search-related components integrated we are going to have a more clear way about how to reuse the same function.

Reply:
I agree with @eduardoformiga. How do you guys feel about adding a task to do that in the next sprint?

Reply:
Agree with you!